tomls/omnibus-arbitrum-mainnet/rewards/seth_perps_liquidations_rewards.toml

```diff
@@ -21,6 +21,7 @@ args = [
         "<%= settings.arb_address %>",
         "<%= settings.usdc_address %>",
         "<%= settings.usde_address %>",
+        "<%= settings.susde_address %>",
```

Comment:
NOTE I am intentionally not registering rewards distributor to catch this with #426 test

Reply:
Good. Tests are failing as expected. Will add missing registrations shortly.


![image](https://github.com/user-attachments/assets/d68ddd1c-239f-4239-aa73-a1cd9369ad7f)